tools/IoTEdgeDevOps/DevOpsLib/BugQueryId.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft. All rights reserved.
+using System.Collections.Generic;
+
+namespace DevOpsLib
+{
+    public static class BugQueryId
+    {
+        public const string SampleQuery = "d7f4e036-0ccb-46a6-a33d-87a932296543";
+
+        public readonly static Dictionary<string, string> QueryNamestoIds = new Dictionary<string, string>() 
+        { 
+            {nameof(SampleQuery), SampleQuery  } 
+        };
```

Comment:
What is the correct convention for an enum of strings? I looked it up, and this was a proposed solution, but I feel there could be a better way. The dictionary is here so that I can cleanly iterate through the properties, but I feel like there is a better way to do this. Repeating the property names in the dict init seems inefficient.

Reply:
1. You don't need to define SampleQuery constant, just define everything in the dictionary should be fine.
2. Do you have different query to find number of bugs of different priority? for example number of P1, P2 bugs?  Bug source from IcM or GitHub or VSTS?